largest-series-product/example.rb

```diff
@@ -17,6 +19,8 @@ def largest_product(length)
   private
 
   def validate_length
+    @length < 0 and
```

Comment:
it's a weird style of failing (using `and`). I got the logic wrong both here and below, clearly I'm not used to it.

I would have just done it with `fail(...) if ...` but I felt I needed to be consistent.